android/src/main/java/io/parity/signer/bottomsheets/PrivateKeyExportBottomSheet.kt

```diff
@@ -0,0 +1,122 @@
+package io.parity.signer.bottomsheets
+
+import android.content.res.Configuration.UI_MODE_NIGHT_NO
+import android.content.res.Configuration.UI_MODE_NIGHT_YES
+import androidx.compose.foundation.*
+import androidx.compose.foundation.layout.*
+import androidx.compose.foundation.shape.RoundedCornerShape
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.Surface
+import androidx.compose.material.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.asImageBitmap
+import androidx.compose.ui.layout.ContentScale
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.platform.LocalInspectionMode
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import androidx.core.graphics.drawable.toBitmap
+import io.parity.signer.R
+import io.parity.signer.components.NetworkCardModel
+import io.parity.signer.components2.KeyCard
+import io.parity.signer.components2.KeyCardModel
+import io.parity.signer.models.EmptyNavigator
+import io.parity.signer.models.Navigator
+import io.parity.signer.models.intoImageBitmap
+import io.parity.signer.ui.theme.*
+import io.parity.signer.uniffi.Action
+import io.parity.signer.uniffi.Address
+
+@Composable
+fun PrivateKeyExportBottomSheet(
+	model: PrivateKeyExportModel,
+	navigator: Navigator,
+) {
+
+//	val bottomSheetScaffoldState = rememberBottomSheetScaffoldState(
+//		bottomSheetState = BottomSheetState(BottomSheetValue.Collapsed)
+//	)
+//	val coroutineScope = rememberCoroutineScope()
```

Comment:
✂️ We shouldn't commit commented code repo, this is fine to stay in git stash if needed for later